tensorboard/plugins/debugger/tf_debugger_dashboard/tf-source-code-view.html

```diff
@@ -419,6 +419,9 @@
               continueToLink.setAttribute('title', 'Continue to');
               // TOOD(cais): Properly style the paper-icon-button.
               continueToLink.addEventListener('tap', () => {
+                this.nodeClicked(
```

Comment:
In the docs for `nodeClicked` above, could you document the third boolean parameter?

Reply:
Done.